lmnet/configs/core/segmentation/segnet_quantize_camvid_tune.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+from easydict import EasyDict
+import tensorflow as tf
+
+from lmnet.common import Tasks
+from lmnet.networks.segmentation.lm_segnet_quantize import LmSegnetQuantize
+from lmnet.datasets.camvid import Camvid
+from lmnet.data_processor import Sequence
+from lmnet.pre_processor import (
+    Resize,
+    DivideBy255,
+)
+from lmnet.data_augmentor import (
+    Brightness,
+    Color,
+    Contrast,
+    FlipLeftRight,
+    Hue,
+)
+from lmnet.quantizations import (
+    binary_mean_scaling_quantizer,
+    linear_mid_tread_half_quantizer,
+)
+from hyperopt import hp
+
+IS_DEBUG = False
+
+NETWORK_CLASS = LmSegnetQuantize
+DATASET_CLASS = Camvid
+
+IMAGE_SIZE = [360, 480]
+BATCH_SIZE = 8
```

Comment:
```suggestion
BATCH_SIZE = 1
```

In my case, `BATCH_SIZE = 1` works well.

Reply:
OK This suggestion is not needed if `DATASET_PREFETCH` is `False` .